Testscripts/Windows/STOR-VHDX-RESIZE-GROWSHRINK.ps1

```diff
@@ -109,8 +109,8 @@ Function Main
 			$controllerType = "IDE"
 			$vmGeneration = Get-VMGeneration $vmName $hvServer
 			if ($vmGeneration -eq 2 ) {
-				$testResult = "SKIPPED"
-				Throw "Generation 2 VM does not support IDE disk, skip test"
+				Write-LogInfo "Generation 2 VM does not support IDE disk, skip test"
+				return "SKIPPED"
```

Comment:
Those 2 lines are not aligned with the existing style in the same script. All set $testresult with values and run throw statement. Your change is return here. It will miss anything needed in finally statement block or later part. Please change.

Reply:
Yes, notice that it does not align existing style.  If set $testResult, and run throw statement, it will be caught by “catch", then it will set the test result as "Fail " and print error. 
When skip case, it should not pop up error message. I have checked the finallly statement block and later part, only do return test result.

Reply:
Yes, I see it has the bug in the catch block. The assignment statement of 'Test fail' should be removed and taken care of it prior to reaching to catch block.

Reply:
Changed based on your comment. Removed ‘Test fail’ in the catch block.